scripts/tests/data/deps_minimum.toml

```diff
@@ -40,7 +40,6 @@ classifiers = [
     'Programming Language :: Python :: 3',
     'Programming Language :: Python :: 3 :: Only',
     'Programming Language :: Python :: 3.8',
```

Comment:
Should Python :: 3.8 be removed as well?

Reply:
Good catch. Yes definitely

Reply:
(This is a testing file so not critical but still useful)